packages/ai/src/ai/modules/task/commands/cmd_task.py

```
@@ -369,160 +366,76 @@ async def on_rrext_store(self, request: Dict[str, Any]) -> Dict[str, Any]:
+    # =========================================================================
+
+    async def _store_fs_open(self, request: Dict[str, Any], args: Dict[str, Any]) -> Dict[str, Any]:
+        """Open a file handle for reading or writing."""
+        fs = self._get_file_store()
+        path = args.get('path')
+        mode = args.get('mode', 'r')
+
+        if mode == 'w':
+            handle_id = await fs.open_write(path, self._connection_id)
+            return self.build_response(request, body={'handle': handle_id})
+        else:
```

Ruff (0.15.7)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)